doc/developer-guides/hld/hld-hypervisor.rst

```diff
@@ -12,6 +12,7 @@ Hypervisor high-level design
    Memory management <hv-memmgt>
    I/O Emulation <hv-io-emulation>
    Physical Interrupt <hv-interrupt>
+   IOC Mediator <hv-ioc-virt>
```

Comment:
@liuyuan-GitHub I took a guess where to put this in the HLD document set... let me know if there's a better place.

Reply:
The location is fine.
By the way, the IOC mediator only work on MRB platform for IVI, the UP2 and NUC do not support IOC hardware and IOC is disabled by default on their boot script.

Reply:
Agree Dave.
We can put it under Developer Guide->Developer Primer, along with Memory Management, I/O Emulation, etc.